docs/tutorials/access-service-across-clusters.md

```diff
@@ -0,0 +1,165 @@
+
+This document provides an example of how to enable MultiClusterService for accessing service across clusters with native service name.
+
+## Prerequisites
+
+### Karmada has been installed
+
+We can install Karmada by referring to [Quick Start](https://github.com/karmada-io/karmada#quick-start), or directly run `hack/local-up-karmada.sh` script which is also used to run our E2E cases.
+
+## Deploy deployment in `member1` cluster
+
+We need to deploy deployment in `member1` cluster:
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+  labels:
+    app: nginx
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - image: nginx
+        name: nginx
+        resources:
+          requests:
+            cpu: 25m
+            memory: 64Mi
+          limits:
+            cpu: 25m
+            memory: 64Mi
+---
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: nginx-propagation
+spec:
+  resourceSelectors:
+    - apiVersion: apps/v1
+      kind: Deployment
+      name: nginx
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+```
+
+After deploying, you can check the created pods:
+```sh
+$ karmadactl get po
+NAME                     CLUSTER   READY   STATUS              RESTARTS   AGE
+nginx-5c54b4855f-6sq9s   member1   0/1     Running             0          28s
+nginx-5c54b4855f-vp948   member1   0/1     Running             0          28s
+```
+
+## Deploy curl pod in `member2` cluster
+
+Let's deploy a curl pod in `member2` cluster:
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: curl
+  labels:
+    app: curl
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: curl
+  template:
+    metadata:
+      labels:
+        app: curl
+    spec:
+      containers:
+      - image: curlimages/curl:latest
+        command: ["sleep", "infinity"]
+        name: curl
+        resources:
+          requests:
+            cpu: 25m
+            memory: 64Mi
+          limits:
+            cpu: 25m
+            memory: 64Mi
+---
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: curl-propagation
+spec:
+  resourceSelectors:
+    - apiVersion: apps/v1
+      kind: Deployment
+      name: curl
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member2
+```
+
+
+After deploying, you can check the created pods:
+```sh
+$ karmadactl get po -C member2
+NAME                    CLUSTER   READY   STATUS    RESTARTS   AGE
+curl-6894f46595-c75rc   member2   1/1     Running   0          15s
+```
+
+Later, we will run the curl command in this pod.
+
+## Deploy MultiClusterService and Service in Karmada
+
+Now, let's deploy Service and MultiClusterService in Karmada:
```

Comment:
Can here briefly describe this place in words? Now we are no need to propagate the service to the member cluster through PropagationPolicy.